doc/classes/OS.xml

```diff
@@ -476,6 +481,7 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether synchronizing the framerate to the monitor's refresh rate is enabled or not.
```

Comment:
Per the docs guidelines, this should follow the pattern:

If [code]true[/code] the framerate is synchronized...